mobile_verifier/src/reward_shares.rs

```diff
@@ -201,11 +207,15 @@ impl MapperShares {
         // the number of subscribers eligible for discovery location rewards
         let discovery_mappers_count = Decimal::from(self.discovery_mapping_shares.len());
 
+        let verified_mapping_event_count = Decimal::from(self.verified_mapping_event_shares.len());
+
         // calculate the total eligible mapping shares for the epoch
         // this could be simplified as every subscriber is awarded the same share
         // however the function is setup to allow the verification mapper shares to be easily
         // added without impacting code structure ( the per share value for those will be different )
-        let total_mapper_shares = discovery_mappers_count * DISCOVERY_MAPPING_SHARES;
+        let total_mapper_shares =
```

Comment:
I believe this is incorrect.  Prior to these changes this code dealt solely with discovery mapping. As per Hip79 each discovery mapping subscriber is allocated 30 reward points.  You would need to get a sum of total shares first for discovery mapping ( as per orig code ) and then a sum of total shares for verification mapping ( sum of all verification mapping rewards for the epoch ). total_mapper_shares then would equal: total_disovery_mapping_shares + total_verification_mapping_shares